googletest/include/gtest/internal/gtest-port-arch.h

```diff
@@ -54,6 +54,9 @@
 #   define GTEST_OS_WINDOWS_PHONE 1
 #  elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
 #   define GTEST_OS_WINDOWS_RT 1
+#  elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_TV_TITLE)
+#   define GTEST_OS_WINDOWS_PHONE 1
```

Comment:
Why are we defining GTEST_OS_WINDOWS_PHONE 1 ? I see that the other definition, GTEST_OS_WINDOWS_TV_TITLE is used in line 1664, gtest.cc but I dont see GTEST_OS_WINDOWS_PHONE used anywhere... could you please explain